queries.sql

```diff
@@ -8,4 +8,71 @@ SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5; 
 SELECT * FROM animals WHERE neutered = TRUE; 
 SELECT * FROM animals WHERE name <> 'Gabumon'; 
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3; 
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3; 
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
```

Comment:
* As stated in the requirement 

> verify that the change was made after updating animals' table 

> verify that change persists after commit, Kindly work on these 🙏

Reply:
@Davidosky007 I have included screenshots for this above. What exactly do you mean by this?

Reply:
According to the requirements you are to write queries that addresses those statements and also add a screenshot of your results

Reply:
I have done that! the second screenshot from the bottom???

Reply:
@mnaibei yes I can see that but you didn't do it also in your transactions

Reply:
@Davidosky007 I do not agree with this review and I have asked for a second opinion.